### PR TITLE
feat: social login and logout

### PR DESCRIPTION
**Pull request by Harshith46**
# Description

implementing FB and google login and logout using bridgekeeper-js library

### CODE REVIEW

File: app/isomorphic/components/molecules/SocialLogin/index.js [hunk 1]
```diff
@@ -1,102 +1,94 @@
...
-  // const twitterKey = get(appKeys, ["twitterAppkey"], "");
+  const [error, setError] = useState("");
+
+  const socialLogin = (e, login) => {
+    e.preventDefault();
+
+    login()
+      .then(() => {
+        checkForMemberUpdated();
+        checkForMemberUpdated().then(res => {
```
Comment (Deokumarjnu) [inline app/isomorphic/components/molecules/SocialLogin/index.js:20]: why we need to call 2 times here, we can just it by line num 20 i think .

File: app/isomorphic/components/molecules/SocialLogin/index.js [hunk 2]
```diff
@@ -1,102 +1,94 @@
...
+      </Button>
+    );
+  };
 
-  // const googleOnClick = (e, serverSideLoginPath) => {
-  //   window.location.href = serverSideLoginPath;
-  // };
+  const GoogleLogin = () => {
+    const { serverSideLoginPath } = withGoogleLogin(
+      "163120650123-m2rj93thcgkfs7js80cop6frppemfo1c.apps.googleusercontent.com",
```
Comment (Deokumarjnu) [inline app/isomorphic/components/molecules/SocialLogin/index.js:56]: can you remove this and take it from config

Comment (Harshith46) [reply app/isomorphic/components/molecules/SocialLogin/index.js:56]: sure

File: app/isomorphic/components/molecules/SocialLogin/index.js (hunk 1 above)
Comment (Harshith46) [reply app/isomorphic/components/molecules/SocialLogin/index.js:20]: sure

**Pull request merged**